src/main/java/Duke/Exception/EmptyDeadlineException.java

```diff
@@ -0,0 +1,4 @@
+package Duke.Exception;
```

Comment:
Names representing packages should be in all lower case. Look at Java coding standards for more details.